crates/snforge-scarb-plugin/src/lib.rs

```diff
@@ -1,4 +1,4 @@
-#![warn(rust_2024_compatibility)]
+#![warn(rust_2021_compatibility)]
```

Comment:
Could you explain this change?

Reply:
It shouldn't be changed. But I'll remove this warning entirely and bump edition to 2024.